projects/github-actions/repo-gardening/src/index.js

```diff
@@ -70,7 +70,15 @@ const automations = [
 
 	debug( `main: Received event = '${ context.eventName }', action = '${ eventPayload.action }'` );
 
+	const taskList = getInput( 'tasks' );
+
 	for ( const { event, action, task } of automations ) {
+		// If the action provided a custom list of tasks to run
+		// and if the task is not one of them, bail.
+		if ( taskList && taskList !== 'all' && ! taskList.split( ',' ).includes( task.name ) ) {
+			continue;
+		}
```

Comment:
Let's do the right thing if someone puts spaces after their commas.
```suggestion
	const taskList = ( getInput( 'tasks' ) || 'all' ).split( ',' ).map( v => v.trim() ) : null;

	for ( const { event, action, task } of automations ) {
		// If the action provided a custom list of tasks to run
		// and if the task is not one of them, bail.
		if ( ! taskList.includes( 'all' ) && ! taskList.includes( task.name ) ) {
			continue;
		}
```